### ISSUE TITLE
[Bug]: Chart does not render*

### ISSUE THREAD

**Issue by taylorjdawson** [SauravKanchan/svelte-chartjs#108]
### Would you like to work on a fix?

- [x] Check this if you would like to implement a PR, we are more than happy to help you go through the process.

### Current and expected behavior

*I understand that you said "if you are experiencing an issue with charts rendering, please create a related issue in [Chart.js repository](https://github.com/chartjs/Chart.js/issues)." However, this code is a direct port from your [scatter example](https://codesandbox.io/s/github/SauravKanchan/svelte-chartjs/tree/master/sandboxes/scatter) but it only works in code-sandbox and not in a local sveltekit repo.

I would expect svelte-chartjs to work in a sveltekit app but I have been unable to discover a working example that uses sveltekit nor have I been able to get mine to work with sveltekit.

### Reproduction

https://github.com/taylorjdawson/svelte-chartjs-example

### chart.js version

4.3.0

### svelte-chartjs version

3.1.2

### Possible solution

Sveltekit uses ssr so that could potentially be causing issues. I am unsure if this library accounts for ssr?

**Comment by Lau-r-a** [comment]
Confirming issue

**Comment by Valentin-Seehausen** [comment]
For SSR you could check out:

https://github.com/Rich-Harris/pancake
or: https://layercake.graphics/

**Comment by harbour0** [comment]
Did anyone find a fix for this? Been unable to even render a line chart within svelte.

**Comment by Sheco** [comment]
Running into the same issue with SvelteKit and svelte-chartjs

**Comment by faulander** [comment]
same here with sveltekit :/

**Comment by dysfunc** [comment]
Is this still a problem for anyone? I'm able to use all the charts in my SvelteKit project.

**Comment by jevb-wsm** [comment]
> Is this still a problem for anyone? I'm able to use all the charts in my SvelteKit project.

Yes :( , even with SSR turned off, charts don't render. Any solution?

**Comment by EdwardLucas2** [comment]
I can't make this work either, it just doesn't work, I can't even get the examples working

**Comment by Guandor** [comment]
Similarly, I can't get even the examples working by directly copy and pasting.

**Comment by tzezar** [comment]
> work fine on sveltekit without this library, I using it.

Could you share some working code example?
